build.gradle

```diff
@@ -80,6 +81,30 @@ targetCompatibility = 1.11
 
 test {
     useJUnitPlatform()
+    finalizedBy(['jacocoTestReport', 'jacocoTestCoverageVerification'])
+}
+
+jacoco {
+    toolVersion = "0.8.5"
+    reportsDir = file("$buildDir/reports/jacoco")
+}
+
+jacocoTestReport {
```

Comment:
✏️ Add a note to the readme about where the coverage report can be found, like we do with the owasp report.